Results_Engine/Query/Max.cs

```diff
@@ -22,22 +22,27 @@
 
 using System;
 using System.Collections.Generic;
+using System.ComponentModel;
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using BH.Engine.Base;
+using BH.oM.Reflection.Attributes;
 using BH.oM.Structure.Results;
 
+
 namespace BH.Engine.Results
 {
     public static partial class Query
     {
         /***************************************************/
         /**** Public Methods                            ****/
         /***************************************************/
-
+        [Description("Determines the maximum force for each component of the BarForce and returns a new BarForce with its concurrent forces.")]
```

Comment:
minor comment on the description here. This is not returning a "new" Bar force, but is just giving the ones with maximum values for all different component. Also, it is always returning a list of length six, not a single new one.

Maybe something like (feel free to reword):

```suggestion
        [Description("Returns a list of the six BarForces that show the maximum value for each force or moment component respectively. The forces returned will have correct concurrent forces for the other component.")]
```

**EDIT**: Something like what you have in the output description I think is clearer.